src/redux/actions/concepts/dictionaryConcepts.js

```diff
@@ -239,6 +255,24 @@ export const addAnswerMappingToConcept = async (url, source, answers) => {
   }
 };
 
+export const addSetMappingToConcept = async (url, source, sets) => {
+  try {
+    const user = localStorage.getItem('username');
+    const mappingUrl = `/users/${user}/sources/${source}/mappings/`;
+    const setMappings = sets.map(set => ({
+      map_type: set.map_type,
+      from_concept_url: url,
+      to_concept_url: set.url,
+    }));
+
+    const promises = setMappings.map(mapping => instance.post(mappingUrl, mapping));
+
+    await axios.all(promises);
+  } catch (error) {
+    notify.show('An error occurred while adding your set mappings', 'error', 3000);
```

Comment:
Do you know what kind of error that this could be? What should the user do in that case?